### PR TITLE
 pkgs/development: switch to fetchFromGitHub part 5

### PR DESCRIPTION
**Pull request by Stunkymonkey**
###### Motivation for this change
use `fetchFromGitHub`

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/vo-amrwbenc/default.nix [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-    sha256 = "85c79997ba7ddb9c95b5ddbe9ea032e27595390f3cbd686ed46a69e485cc053c";
+
+  src = fetchFromGitHub {
+    owner = "mstorsjo";
+    repo = "vo-amrwbenc";
+    rev = "v${version}";
+    sha256 = "sha256-oHhoJAI47VqBGk9cO3G5oqnHpWxA2jnJs103MwcYj+w=";
   };
 
+  buildInputs = [ autoreconfHook ];
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/vo-amrwbenc/default.nix:14]:
```suggestion
  nativeBuildInputs = [ autoreconfHook ];
```

File: pkgs/development/libraries/luabind/default.nix [hunk 2]
```diff
@@ -1,11 +1,13 @@
-{lib, stdenv, fetchurl, boost-build, lua, boost}:
+{ lib, stdenv, fetchFromGitHub, boost-build, lua, boost }:
 
 stdenv.mkDerivation {
   name = "luabind-0.9.1";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/luabind/default.nix:5]:
```suggestion
  pname = "luabind";
  version = "0.9.1";

```

File: pkgs/development/libraries/luabind/default.nix [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
 
-  src = fetchurl {
-    url = "https://github.com/luabind/luabind/archive/v0.9.1.tar.gz";
-    sha256 = "0e5ead50a07668d29888f2fa6f53220f900c886e46a2c99c7e8656842f05ff2d";
+  src = fetchFromGitHub {
+    owner = "luabind";
+    repo = "luabind";
+    rev = "v0.9.1";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/luabind/default.nix:9]:
```suggestion
    rev = "v${version}";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by Stunkymonkey** [comment]
should i rebase this to "staging"?

**Comment by SuperSandro2000** [comment]
Now that we only have below 100 rebuilds we can target master.

File: pkgs/development/libraries/nco/default.nix [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
 stdenv.mkDerivation rec {
   pname = "nco";
   version = "5.0.3";
 
-  src = fetchzip {
-    url = "https://github.com/nco/nco/archive/${version}.tar.gz";
+  src = fetchFromGitHub {
+    owner = "nco";
+    repo = "nco";
+    rev = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/nco/default.nix:10]:
```suggestion
    rev = version;
```

File: pkgs/development/libraries/luabind/default.nix [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
+  pname = "luabind";
+  version = "0.9.1";
 
-  src = fetchurl {
-    url = "https://github.com/luabind/luabind/archive/v0.9.1.tar.gz";
-    sha256 = "0e5ead50a07668d29888f2fa6f53220f900c886e46a2c99c7e8656842f05ff2d";
+  src = fetchFromGitHub {
+    owner = "luabind";
+    repo = "luabind";
+    rev = "v${version}";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/luabind/default.nix:10]:
```suggestion
    rev = version;
```

**Pull request merged**